server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/ImapStepdefs.java

```diff
@@ -99,17 +99,16 @@ public void hasNoMessageInMailbox(String mailbox) throws Throwable {
         }
     }
 
+    @SuppressWarnings("resource")
     @Given("^the user has an open IMAP connection with mailbox \"([^\"]*)\" selected")
     public void openImapConnectionAndSelectMailbox(String mailbox) throws Throwable {
         String login = userStepdefs.getConnectedUser();
         String password = userStepdefs.getUserPassword(login);
 
-        try (IMAPMessageReader imapMessageReader = new IMAPMessageReader()) {
-            imapConnections.put(mailbox, imapMessageReader
-                .connect(LOCALHOST, IMAP_PORT)
-                .login(login, password)
-                .select(mailbox));
-        }
+        imapConnections.put(mailbox, new IMAPMessageReader()
+            .connect(LOCALHOST, IMAP_PORT)
+            .login(login, password)
+            .select(mailbox));
```

Comment:
do we close connexions in a after or something like that?

Reply:
Not sure

Reply:
Nope

Reply:
could be a good move :) warnings are not just to bore us